### PR TITLE
Bug 1799736 - Use title of Github Pull Request instead of generic "GitHub Pull Request" string

### PR DESCRIPTION
**Pull request by dklawren**
Simple fix.

### CODE REVIEW

**Review by cgsheeh — APPROVED** [review]

File: Bugzilla/API/V1/Github.pm [hunk 1]
```diff
@@ -99,7 +99,7 @@ sub pull_request {
     bug         => $bug,
     creation_ts => $timestamp,
     data        => $html_url,
-    description => 'GitHub Pull Request',
+    description => $title,
```
Comment (jonalmeida) [inline Bugzilla/API/V1/Github.pm:102]: I didn't realize that the `pr_number` attribute was also available when I filed this bug, which might be useful to have there too?

```suggestion
    description => "$pr_number - $title",
```

Comment (JohanLorenzo) [reply Bugzilla/API/V1/Github.pm:102]: Thanks for the super quick fix @dklawren! I love the fact I won't have to manually attach pull requests anymore ❤️ 

As a release engineer, I often deal with changes across multiple repos just to accomplish a single thing (for instance [bug 1786780](https://bugzilla.mozilla.org/show_bug.cgi?id=1786780)). I wouldn't be surprised if other teams have a similar workflow.

It'd be super helpful if the repo name is part of the description too. E.g.: 

```suggestion
    description => "[$repo_name] $title ($pr_number)",
```

with `$repo_name` being[1]: 

```
my $repo_name = $payload->{repository}->{name};
```

[1] https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request

Comment (jcristau) [reply Bugzilla/API/V1/Github.pm:102]: There are cases where a single bug gets PR attachments from multiple github repos, in which case the pr number is maybe also confusing?  Not sure, maybe this doesn't happen often enough to be a problem.

Comment (dklawren) [reply Bugzilla/API/V1/Github.pm:102]: > Thanks for the super quick fix @dklawren! I love the fact I won't have to manually attach pull requests anymore heart
> 
> As a release engineer, I often deal with changes across multiple repos just to accomplish a single thing (for instance [bug 1786780](https://bugzilla.mozilla.org/show_bug.cgi?id=1786780)). I wouldn't be surprised if other teams have a similar workflow.
> 
> It'd be super helpful if the repo name is part of the description too. E.g.:
> 
> with `$repo_name` being[1]:
> 
> ```
> my $repo_name = $payload->{repository}->{name};
> ```

Sounds like a great suggestion. Do you mind filing a new bug about adding the missing values in and we can go ahead and merge this one for now. If you file a bug I can get the new changes out next week,

Comment (JohanLorenzo) [reply Bugzilla/API/V1/Github.pm:102]: Sure! I just filed [bug 1799924](https://bugzilla.mozilla.org/show_bug.cgi?id=1799924)

**Pull request merged**